frontend/src/stimulus/controllers/dynamic/filter/filters-form.controller.ts

```diff
@@ -142,6 +142,11 @@ export default class FiltersFormController extends Controller {
     this.displayFiltersValue = !this.displayFiltersValue;
   }
 
+  showDisplayFilters() {
+    this.displayFiltersValue = true;
+    this.displayFiltersValueChanged();
```

Comment:
Is this actually necessary? IIRC when these are declared Stimulus controller values, they are automatically observed and call the `valueNameValueChanged` without having to invoke it.

Reply:
I know what you mean. I had the same assumption initially and was surprised by this behavior as well.

To be sure, I tried commenting out the call to `showDisplayFilters` again -> the filter component is not toggled automatically when this line is missing.